webapp/tests/karma/ts/test-utils/test-setup.ts

```diff
@@ -0,0 +1,18 @@
+import { TestBed, TestModuleMetadata } from '@angular/core/testing';
+import { MaterialTestModule } from './material-test.module';
+
+/**
+ * Helper function to configure testing modules with proper Material icon support
+ * to prevent the ":icon-back" error in unit tests
+ */
+export const configureTestingModule = (config: TestModuleMetadata): void => {
+  // Ensure imports array exists
+  config.imports = config.imports || [];
+  
+  // Add MaterialTestModule if it doesn't already exist
+  if (!config.imports.includes(MaterialTestModule)) {
```

Comment:
I'm still seeing some :icon-back errors in tests:
https://github.com/medic/cht-core/actions/runs/14491214068/job/40652590527?pr=9902#step:10:6669
https://github.com/medic/cht-core/actions/runs/14491214068/job/40652590527?pr=9902#step:10:6782

Is it because these files were not updated or?